src/platform_impl/macos/accelerator.rs

```diff
@@ -0,0 +1,112 @@
+use cocoa::appkit::NSEventModifierFlags;
+
+/// Mnemonic is deprecated since macOS 10
+pub fn remove_mnemonic(string: impl AsRef<str>) -> String {
+    string
+        .as_ref()
+        .replace("&&", "[~~]")
+        .replace("&", "")
+        .replace("[~~]", "&")
+}
```

Comment:
```suggestion
/// Mnemonic is deprecated since macOS 10
pub fn remove_mnemonic(string: impl AsRef<str>) -> String {
    string
        .as_ref()
        .replace("&", "")
}
```

Reply:
Hmm, what if we want to have "&" in the label like "Save & Close"